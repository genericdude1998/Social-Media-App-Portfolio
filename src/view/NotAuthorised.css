*{
    box-sizing: border-box;
    margin: 0px;
}
.notAuth{
    font-size: 3vw;
    color: white;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    margin: 10vw;
    height: 50vh;
    background-color: royalblue;
}

.link{
    text-decoration: none;
    font-weight: bold;
}