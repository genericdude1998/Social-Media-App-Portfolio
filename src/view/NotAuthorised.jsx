import React from 'react';
import { Link } from 'react-router-dom';
import styles from './NotAuthorised.css';

const NotAuthorised = () => {
    return (
        <div className={styles.notAuth}>
            You do not have the required authorisation.
            Please go back to <Link to={'/'} className={styles.link}>Login</Link>
        </div>
    );
}

export default NotAuthorised;
