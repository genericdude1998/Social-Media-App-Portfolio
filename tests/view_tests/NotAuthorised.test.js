import React from 'react';
import { shallow } from 'enzyme';
import NotAuthorised from '../../src/view/NotAuthorised';

describe('NotAuthorised', () => {
    it('should match snap', () => {
        const wrapper = shallow(<NotAuthorised />);
        expect(wrapper).toMatchSnapshot();
    });
});