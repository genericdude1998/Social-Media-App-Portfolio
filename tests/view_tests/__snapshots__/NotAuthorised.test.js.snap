// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotAuthorised should match snap 1`] = `
<div
  className="notAuth"
>
  You do not have the required authorisation. Please go back to 
  <Link
    className="link"
    to="/"
  >
    Login
  </Link>
</div>
`;
